### ISSUE TITLE
implement local storage : images and tree galery

### ISSUE THREAD

**Issue by thanhnh-infinity** [phylotastic/Pt_Mobile_Application#40]

**Comment by arlin** [comment]
@thanhnh-infinity, are you working on this?   Let's discuss with @epontell whether this is a priority, OK?

**Comment by thanhnh-infinity** [comment]
Yes, Dr Arlin. This issues refers to store tree and images that are taken from users on their devices. I will discuss with Dr Pontelli before I start working on it. Right now, it has not implemented yet.
